sci-geosciences/gmt/gmt-4.5.17.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF=yes
+
+inherit autotools-utils multilib
```

Comment:
`autotools-utils` is banned in modern EAPIs, do not use this

Reply:
Replaced on autotools